packages/gafl-webapp-service/src/server.js

```diff
@@ -63,8 +70,8 @@ const plugIns = [
        * 31 of the GDS template. This will probably come up as an advisory in the PEN test.
        */
       fontSrc: ['self', 'fonts.gstatic.com', 'data:'],
-      scriptSrc: ['self', 'unsafe-inline'],
-      generateNonces: false
+      scriptSrc: ["'sha256-+6WnXIl4mbFTCARd8N3COQmT3bJJmo32N8q8ZSQAIcU='"],
```

Comment:
Can we extract this to a descriptive variable name pls?

Reply:
removed